api-spec/vela-server-v0.5.0.json

```diff
@@ -1,4867 +1,5264 @@
-          "x-go-name": "Runtime"
-        },
-        "stage": {
-          "type": "string",
-          "x-go-name": "Stage"
-        },
-        "started": {
-          "type": "integer",
-          "format": "int64",
-          "x-go-name": "Started"
-        },
-        "status": {
-          "type": "string",
-          "x-go-name": "Status"
-        }
-      },
-      "x-go-package": "github.com/go-vela/types/library"
-    },
-    "User": {
-      "type": "object",
-      "title": "User is the library representation of a user.",
-      "properties": {
-        "active": {
-          "type": "boolean",
-          "x-go-name": "Active"
-        },
-        "admin": {
-          "type": "boolean",
-          "x-go-name": "Admin"
-        },
-        "favorites": {
-          "type": "array",
-          "items": {
-            "type": "string"
-          },
-          "x-go-name": "Favorites"
-        },
-        "id": {
-          "type": "integer",
-          "format": "int64",
-          "x-go-name": "ID"
-        },
-        "name": {
-          "type": "string",
-          "x-go-name": "Name"
-        },
-        "token": {
-          "type": "string",
-          "x-go-name": "Token"
-        }
-      },
-      "x-go-package": "github.com/go-vela/types/library"
-    },
-    "Webhook": {
-      "type": "object",
-      "title": "Hook is the library representation of a webhook for a repo.",
-      "properties": {
-        "branch": {
-          "type": "string",
-          "x-go-name": "Branch"
-        },
-        "build_id": {
-          "type": "integer",
-          "format": "int64",
-          "x-go-name": "BuildID"
-        },
-        "created": {
-          "type": "integer",
-          "format": "int64",
-          "x-go-name": "Created"
-        },
-        "error": {
-          "type": "string",
-          "x-go-name": "Error"
-        },
-        "event": {
-          "type": "string",
-          "x-go-name": "Event"
-        },
-        "host": {
-          "type": "string",
-          "x-go-name": "Host"
-        },
-        "id": {
-          "type": "integer",
-          "format": "int64",
-          "x-go-name": "ID"
-        },
-        "link": {
-          "type": "string",
-          "x-go-name": "Link"
-        },
-        "number": {
-          "type": "integer",
-          "format": "int64",
-          "x-go-name": "Number"
-        },
-        "repo_id": {
-          "type": "integer",
-          "format": "int64",
-          "x-go-name": "RepoID"
-        },
-        "source_id": {
-          "type": "string",
-          "x-go-name": "SourceID"
-        },
-        "status": {
-          "type": "string",
-          "x-go-name": "Status"
-        }
-      },
-      "x-go-name": "Hook",
-      "x-go-package": "github.com/go-vela/types/library"
-    }
-  },
-  "securityDefinitions": {
-    "ApiKeyAuth": {
-      "type": "apiKey",
-      "name": "Authorization",
-      "in": "header"
-    }
-  }
-}
+{
```

Comment:
Can we drop the version from the file name?

Reply:
Updated.